### PR TITLE
Add accordion hours

### PR DESCRIPTION
**Pull request by DominikDR**

### CODE REVIEW

File: weatherApp.js [hunk 1]
```diff
@@ -122,7 +124,9 @@ windButton.addEventListener("click", renderWindContent);
 
 //zapukać do api i połączyć się z nim za pomocą XHR i promisów. Ew async await
 //zrobić listę danych: Miasto(nagłówek), niżej prognoza pogody na dziś, jutro, pojutrze w formacie `data-"rainy"`
-
+const upperFirstLetter = (name) => {
```
Comment (Dammic) [inline weatherApp.js]: From the way this param is named, I expect a single element. However, below you call name[0] suggesting it's an array.
This function should be called with a single string element, judging from the name (you call upper on string).

File: weatherApp.js [hunk 2]
```diff
@@ -122,7 +124,9 @@ windButton.addEventListener("click", renderWindContent);
...
+  return `${name[0].toUpperCase()}${name.slice(1)}`
```
Comment (Dammic) [inline weatherApp.js]: You can probably do that with only css:
https://css-tricks.com/almanac/selectors/f/first-letter/

File: weatherApp.js [hunk 3]
```diff
@@ -192,26 +196,59 @@ const handleDayClick = (day, divDay) => {
...
 		return divHourBox;
 	})
 	return hourAndTempList;
 }
 
+/*const createAccordionContent = (day, parentHourDiv) => {
+	console.log("day", day);
+	const accordionDiv = `dupadupadpa`
+	return parentHourDiv.insertAdjacentHTML('beforeend',`<div>${accordionDiv}</div>`)
+}*/
```
Comment (Dammic) [inline weatherApp.js]: Unnecessary comment

Comment (DominikDR) [reply weatherApp.js]: Ok, I had doubts if it wouldn't be better to use another function for creating accordion content.

File: weatherApp.js (hunk 2 above)
Comment (DominikDR) [reply weatherApp.js]: Done. It works, thx :)

**Pull request merged**